src/components/FilterPanel/FilterPanelCheckboxWithOverflowMenu/__tests__/FilterPanelCheckboxWithOverflowMenu.spec.js

```diff
@@ -0,0 +1,260 @@
+/**
+ * @file Filter panel checkbox with overflow menu tests.
+ * @copyright IBM Security 2020
+ */
+
+import React from 'react';
+import {
+  render,
+  fireEvent,
+  waitForElementToBeRemoved,
+  screen,
+} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import FilterPanelCheckboxWithOverflowMenu from '../FilterPanelCheckboxWithOverflowMenu';
+import OverflowMenuItem from '../../../OverflowMenuItem';
+
+describe(FilterPanelCheckboxWithOverflowMenu.name, () => {
+  test('should have no Axe or DAP violations', async () => {
+    const main = document.createElement('main');
+    render(
+      <FilterPanelCheckboxWithOverflowMenu
+        labelText="test checkbox"
+        id="test-checkbox-id"
+      />,
+      {
+        // DAP requires a landmark '<main>' in the DOM:
+        container: document.body.appendChild(main),
+      }
+    );
+    await expect(document.body).toHaveNoAxeViolations();
+    await expect(document.body).toHaveNoDAPViolations(
+      `${FilterPanelCheckboxWithOverflowMenu.name}-default`
+    );
+  });
+
+  test('should have no Axe or DAP violations', async () => {
+    const main = document.createElement('main');
+    render(
+      <FilterPanelCheckboxWithOverflowMenu
+        labelText="checkbox label"
+        overflowMenuAriaLabel="filter selection options"
+        id="checkbox-id"
+      >
+        <OverflowMenuItem primaryFocus itemText="option 1" />
+        <OverflowMenuItem itemText="option 2" />
+      </FilterPanelCheckboxWithOverflowMenu>,
+      {
+        // DAP requires a landmark '<main>' in the DOM:
+        container: document.body.appendChild(main),
+      }
+    );
+
+    // Open overflow menu and waitFor for options to appear on the DOM.
+    fireEvent.mouseEnter(screen.getByLabelText(/checkbox label/i));
+    userEvent.click(screen.getByLabelText(/filter selection options/i));
+    await screen.findByText(/option 1/i);
+
+    await expect(document.body).toHaveNoAxeViolations();
+    await expect(document.body).toHaveNoDAPViolations(
```

Comment:
@jendowns, I wonder if you have any insights on this. This test is failing and I _think_ it has to do with the `OverflowMenu` implementation from Carbon that is maybe the cause?